usr/bin/irdata_parse_bar.sh

```diff
@@ -43,18 +43,11 @@ if [ ! "$DATADIR" ] || [ ! "$FILENAMES" ]; then
 fi
 
 
-#it is assumed that all records in the virushost file refer to protein-protein interactions
-#it is not possible to check for these (as done in the below commented code) because virushost
-#does not follow regular mitab conventions and interactor type is not provided
-#should this change in future, the following code could be uncommented
```

Comment:
Is this no longer true?

Reply:
This file most likely started as a copy of `irdata_parse_virushost.sh`. There the comment does make sense. For the BAR data source this comment is misleading.